main/guides/orchestration/getting-started/contract-walkthrough/cross-chain-unbond.md

```diff
@@ -178,16 +152,17 @@ const makeUnbondAndLiquidStakeInvitation = () =>
         exit: M.any(),
       }),
     );
-```
-
-## Public Facet
-```javascript
-const publicFacet = Far('SwapAndStake Public Facet', {
-    makeUnbondAndLiquidStakeInvitation,
+  },
 });
 
 return harden({ publicFacet });
-};
 ```
 
-Defines the `publicFacet` for the contract, which includes the method to make an `invitation`, and returns the hardened public facet.
\ No newline at end of file
+Defines the `publicFacet` for the contract, which includes the method to make an `invitation`, and returns the hardened public facet. Defining `publicFacet` with `zone.exo` makes it [remotely accessible](/glossary/#exo) and persistent through contract upgrades with a [durable `zone`](/glossary/#zone).
+
+## `start` Function
+```javascript
+export const start = withOrchestration(contract);
+```
+
+Defines the `start` function of the contract as a wrapped veriosn of [`contract` function above](#contract-function) with `withOrchestration`.
```

Comment:
Can we rewrite this sentence, because it seems weird for some reason with `as a wrapped...`.

Reply:
Done.